gunicorn/_compat.py

```diff
@@ -3,7 +3,7 @@
 from gunicorn import six
 
 PY26 = (sys.version_info[:2] == (2, 6))
-PY33 = (sys.version_info >= (3, 3))
+PY3 = (sys.version_info[0] >= 3)
```

Comment:
We already use `PY3` from `gunicorn.six` module (see line 66 of this file for example) so I think we can delete the duplicated `PY3`.